.env.docker.development

```
@@ -0,0 +1,19 @@
+# Docker Environment Configuration for development
+# Generated on 2025-09-12T22:47:44.332Z
+
+NODE_ENV=development
+PORT=3000
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/binaai_dev
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)